husky_bringup/launch/dual_ur5_arm_config/husky_dual_ur5_planning_execution.launch

```diff
@@ -0,0 +1,13 @@
+<launch>
```

Comment:
this launch is unused.

Guessing it's robot/sim generic, so it should be in husky_control either way

Probably generalizable to single ur5 case as well